shared/test/test_net_sim_api.rb

```diff
@@ -0,0 +1,55 @@
+require 'minitest/autorun'
+require 'rack/test'
+require File.expand_path '../../../deployment', __FILE__
+require File.expand_path '../../middleware/net_sim_api', __FILE__
+
+ENV['RACK_ENV'] = 'test'
+
+class NetSimApiTest < Minitest::Unit::TestCase
+
+  def create_record(record)
```

Comment:
Perhaps make these helpers private and move them to the end with a comment so that it is obvious they are helpers.